packages/updatePrescriptionStatus/src/updatePrescriptionStatus.ts

```diff
@@ -0,0 +1,160 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import {APIGatewayProxyEvent, APIGatewayProxyResult} from "aws-lambda"
+import {Logger} from "@aws-lambda-powertools/logger"
+import {injectLambdaContext} from "@aws-lambda-powertools/logger/middleware"
+import {DynamoDBClient, PutItemCommand} from "@aws-sdk/client-dynamodb"
+import {marshall} from "@aws-sdk/util-dynamodb"
+import middy from "@middy/core"
+import inputOutputLogger from "@middy/input-output-logger"
+import errorHandler from "@nhs/fhir-middy-error-handler"
+import {v4 as uuidv4} from "uuid"
+
+const logger = new Logger({serviceName: "updatePrescriptionStatus"})
+const client = new DynamoDBClient({region: "eu-west-2"})
+const tableName = process.env.TABLE_NAME
+
+const MISSING_FIELDS_RESPONSE: APIGatewayProxyResult = {
+  statusCode: 400,
```

Comment:
All responses should be a valid fhir response, this should be some sort of operation outcome.

Reply:
Also the error message is a little broad, we could provide back to the client which fields are missing/invalid etc, see comments further down.

Reply:
Validation will be implemented in another ticket.